docs/products/redis/get-started.rst

```diff
@@ -1,59 +1,38 @@
-Get Started with Aiven for Redis
+Get started with Aiven for Redis
 ================================
 
-Get up and running with Aiven for Redis. This article shows you how to set up a Redis service in Aiven and learn more about the platform.
+Aiven for Redis services are managed from the `Aiven
+Console <https://console.aiven.io/>`__ .
 
-Create a new Aiven service
---------------------------
+You can start an Aiven for Redis service by following the steps laid out
```

Comment:
You're relying later that users will follow the instructions, I'd omit "You can " and use "Create an Aiven for Redis service by following the steps laid out"